packages/env/lib/create-docker-compose-config.js

```diff
@@ -0,0 +1,55 @@
+module.exports = function createDockerComposeConfig(
+	pluginPath,
+	pluginName,
+	pluginTestsPath
+) {
+	const commonVolumes = `
+      - ${ pluginPath }/:/var/www/html/wp-content/plugins/${ pluginName }/
+      - ${ pluginPath }${ pluginTestsPath }/e2e-tests/mu-plugins/:/var/www/html/wp-content/mu-plugins/
+      - ${ pluginPath }${ pluginTestsPath }/e2e-tests/plugins/:/var/www/html/wp-content/plugins/${ pluginName }-test-plugins/`;
+	const volumes = `
+      - ${ pluginPath }/../${ pluginName }-wordpress/:/var/www/html/${ commonVolumes }`;
+	const testsVolumes = `
+      - tests-wordpress:/var/www/html/${ commonVolumes }`;
+	return `version: '2'
```

Comment:
For what reason do we use an older version?

Prior to all the revisions to the Docker worfklow, we ran at v3.1

https://github.com/WordPress/gutenberg/blob/v6.4.0/docker-compose.yml#L1

Reply:
Some of the features we were using broke with version 3 and https://www.npmjs.com/package/docker-compose.